os_migrate/roles/validate_resource_files/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: validate resource files
+  os_migrate.os_migrate.validate_resource_files:
+    paths: 
+        - "{{ resource_files }}"
+  register: "{{ resource_files_validation }}"
+  when: resource_files_validate_file
```

Comment:
We could rename to `os_migrate_validate_files`, put it into `roles/validate_resource_files/defaults/main.yml` and delete all variable defaults of the pattern `import_workloads_validate_file` etc.